qa/vagrant/build.gradle

```diff
@@ -92,6 +71,24 @@ repositories {
 }
 
 dependencies {
+  String maybeTestsSeed = System.getProperty("tests.seed", null);
+  if (maybeTestsSeed != null) {
+    seed = new BigInteger(maybeTestsSeed, 16).longValue()
+    formattedSeed = maybeTestsSeed
+  } else {
+    seed = new Random().nextLong()
+    formattedSeed = String.format("%016X", seed)
+  }
+
+  String maybeUpdradeFromVersions = System.getProperty("tests.packaging.upgrade.from.versions", null)
+  if (maybeUpdradeFromVersions != null) {
+    upgradeFromVersions = maybeUpdradeFromVersions.split(",")
+  } else {
+    upgradeFromVersions = new File(project.projectDir, 'src/main/resources/versions')
```

Comment:
Nik is right, but I think his idea might work to delay until we need to resolve.